drivers/volume/portworx/schedops/k8s-schedops.go

```diff
@@ -534,6 +534,32 @@ func (k *k8sSchedOps) IsPXEnabled(n node.Node) (bool, error) {
 	return true, nil
 }
 
+// GetStorageInfo returns cluster pair info from destination cluster refereced by kubeconfig
+func (k *k8sSchedOps) GetRemotePXNodes(destKubeConfig string) ([]node.Node, error) {
```

Comment:
Wrong comment for function. Also, this is always returning one node. So can be called GetRemotePXNode and can return just node.Node instead of an array.
Or, return all the nodes from the remote cluster.

Reply:
that was earlier plan to get all px nodes for future node